Install uuid, create delete alert, create a header section and a note…

chayboats opened the pull request:
… section, update styles for dropdown
Install uuid, 
create delete alert, 
create a header section and a note section, 
update styles for dropdown

chayboats committed:
Install uuid, create delete alert, create a header section and a note section, update styles for dropdown

wdonray commented on pages/home/index.js:
```diff
@@ -1,107 +1,40 @@
-        inputTitleClass={showForm ? "input-text" : "hide"}
-        buttonTitleClass={showForm ? "add-note" : "hide"}
+        inputTitleClass={showForm ? 'input-text' : 'hide'}
+        buttonClass={showForm ? 'add-note' : 'hide'}
```
**suggestion:** I would suggest to pass down a prop that represents if the form is shown and then toggle between classes in the component. This reduces the amount of props and makes it more clear.

wdonray commented on pages/home/internals/components/DeleteAlert.js:
```diff
@@ -1,10 +1,26 @@
+        >
+          Delete
+        </button>
+      </div>
```
**nitpick:** Make this a form with type submit on the delete button.

wdonray commented on pages/home/internals/components/Note-Section/Note.js:
```diff
@@ -0,0 +1,35 @@
+          className="fa-solid fa-ellipsis-vertical"
+        ></i>
+      </span>
+      <div className={'dropdown-container'}>
```
**nitpick:** Brackets are not needed here.

wdonray commented on pages/home/internals/components/Note-Section/Note.js:
```diff
@@ -0,0 +1,35 @@
+        ></i>
+      </span>
+      <div className={'dropdown-container'}>
+        <div className={isSelected ? 'note-dropdown' : 'hide'}>
```
**note:** This example is perfect for reference of my suggestion from above.

chayboats committed:
Create formIsDisplayed prop to pass the state of showForm to the NoteSection.
Change delete alert to a form with a type submit delete button.
Remove unneeded brackets

chayboats committed:
Rename prop to isFormDisplayed
Exchange onClick to onSubmit.
Add type button to cancel button.
